packages/web-forms/src/components/controls/RankControl.vue

```diff
@@ -164,18 +182,25 @@ const swapItems = (index: number, newPosition: number) => {
 @import 'primeflex/core/_variables.scss';
 
 // Variable definition to root element
-.rank-control {
+.range-control-container {
```

Comment:
This section defines CSS variables specific to this component, helping us identify patterns across question types that can later be extracted at a higher level for reuse. We'll get to this when adjusting the styles of Web Forms once PrimeVue is upgraded.